packages/salesforcedx-apex-replay-debugger/test/integration/adapter.test.ts

```diff
@@ -148,9 +148,8 @@ describe('Replay debugger adapter - integration', function() {
         classBPath,
         classBValidLines
       );
-
+      // tslint:disable-next-line:no-floating-promises
```

Comment:
Could you provide some context on why you choose to ignore the next line?

Reply:
I still don't know why this line can be ignored safely, other than the fact that this is just in the test so a floating promise is less catastrophic. Could you provide more details.